src/analyzer/analyzerebur128mit.h

```diff
@@ -0,0 +1,27 @@
+#ifndef ANALYZEREBUR128MIT_H_
+#define ANALYZEREBUR128MIT_H_
+
+#include "analyzer/analyzer.h"
+#include "preferences/usersettings.h"
+#include <ebur128.h>
```

Comment:
forward declaration is difficult in this case of an anonymous struct typedef, so I prefer to keep this include.